web/app/components/Chat/Chat.jsx

```diff
@@ -329,21 +329,25 @@ class Chat extends React.Component {
     _onTipSuccess() {
         let tip = this._parseTip();
         let message = {
-            user: "SYSTEM",
+            user: "TROLLBOT",
             message: this.props.currentAccount + " tipped " + tip.to + " " + tip.amount + " " + tip.asset,
             color: "#B71A00"
         };
 
         // Public and local broadcast
-        this._broadCastMessage(message);
+        this._broadCastMessage(message, local = false);
```

Comment:
I't wasnt working for me yesterday... I should test better before trying to "fix" something. All because chat history works not so good.  I understand (not sure) that history is taken from first random peer, but some peers have more history than others. In consequence,  if I'm really unlucky my history count is 0, no messages at all. This can perhaps work if trollbox being crowded, but for now it sucks. Would be very cool if I could read last 100 messages, not just 2. Something like if history < 100 {search peer with longest history} would be cool.